Paws4Me/Paws4Me/Display All Adoptable Pets/View Controllers/AllPetDetailViewController.swift

```diff
@@ -21,10 +21,12 @@ class AllPetDetailViewController: UIViewController {
     private lazy var petDataViewModel = AllPetDataViewModel(repository: PetDataRepository(),
                                                             delegate: self)
     private var searchBarController = UISearchBar()
+    fileprivate var petIconButtonArray: [UIButton] = []
```

Comment:
Any specific reason you want this to be `fileprivate`?

Reply:
There is not a reason as I only need to access this variable inside this containing file. Thus this should be private. I will amend it now.